.github/workflows/autorelease.yml

```diff
@@ -74,3 +74,12 @@ jobs:
           echo gh release upload "${tag}" "${msi}"
         env:
           GITHUB_TOKEN: ${{ secrets.GH_PAT }}
+      -
+        name: "Upload deb files to apt hosting"
+        run: |
+          for D in dist/*.deb; do
+            curl -H"X-Filename: ${D}" -H'X-Apikey: ${APIKEY}' -XPOST --data-binary @$D http://packages.gauner.org/repos/gopass/upload
```

Comment:
The only thing I would complain about is that it's going over a plain HTTP line instead of HTTPS.

Reply:
Yes. But this will take some time to resolve. That's why I said they it's not ready for public consumption.

But for a test it should be alright.

Once I have the infrastructure in place I'll wipe the repo and regenerate the signing key.